### PR TITLE
fix(Checkout): Points initialization

### PR DESCRIPTION
**Pull request by itsDiwaker**
initialize point to apply to order total or available participant's point balance.

#355

### CODE REVIEW

**Comment by itsDiwaker** [comment]
please check #355 for queries.

File: templates/checkout/shipping.liquid [hunk 1]
```diff
@@ -218,7 +217,21 @@
                                 <div class="grid-x grid-padding-x flex-container margin-bottom">
                                     <div class="cell small-8">
                                         <div class="input-group">
-                                            <input type="text" class="input-group-field border-color-medium-gray no-border-right js-points-applied">
+
+                                            {% assign applicablePoints = 0 %}
+
+                                            {% if websiteSettings.storefrontPayments.uom != 'none' %}
+                                                {% capture applicablePoints %}{{order.total | ceil | times: websiteSettings.storefrontPayments.pointConversion}}{% endcapture %}
```
Comment (CristinaSolana) [inline templates/checkout/shipping.liquid]: @itsDiwaker ceil should happen after times filter. This has been fixed in several places on another open PR.

Comment (itsDiwaker) [reply templates/checkout/shipping.liquid]: just copied it from `checkout_order_summary_detail.liquid`. I think I should fix there too.

**Comment by CristinaSolana** [comment]
@itsDiwaker please resolve conflicts and I can merge, this LGTM at the moment.

**Comment by itsDiwaker** [comment]
@CristinaSolana  conflict resolved!

**Pull request merged**